i18n/en-US.yml

```diff
@@ -49,7 +420,51 @@ common:
     drive: Drive
     micromobility: E-Scooter
     micromobilityRent: Rental E-Scooter
-    walk: Walk 
+    walk: Walk
+  daysOfWeek:
+    monday: Monday
+    tuesday: Tuesday
+    wednesday: Wednesday
+    thursday: Thursday
+    friday: Friday
+    saturday: Saturday
+    sunday: Sunday
+  daysOfWeekCompact:
+    monday: Mon.
+    tuesday: Tue.
+    wednesday: Wed.
+    thursday: Thu.
+    friday: Fri.
+    saturday: Sat.
+    sunday: Sun.
+  daysOfWeekPlural:
+    monday: Mondays
+    tuesday: Tuesdays
+    wednesday: Wednesdays
+    thursday: Thursdays
+    friday: Fridays
+    saturday: Saturdays
+    sunday: Sundays
+  # Common form UI messages
+  # Note to translator: these values are width-constrained.
+  forms:
+    back: Back
+    cancel: Cancel
+    defaultValue: "{value} (default)"
+    delete: Delete
+    edit: Edit
+    finish: Finish
+    next: Next
+    no: No
+    save: Save
+    yes: Yes
+  # Shared itinerary description messages
+  itineraryDescriptions:
+    calories: Cal
+    departure: DEPARTURE
+    status: STATUS
```

Comment:
@philip-cline These table headings should be first letter capitalized here. To render them all-caps, there should be some CSS for the corresponding elements in `viewers.css`.

Reply:
IMO, `departure` and `status` should be under `components.PatternRow`, or is there a reason to place them under `common`?

Reply:
Both of these addressed in https://github.com/opentripplanner/otp-react-redux/pull/408/commits/8a8cfaef8109184eb3f7d63cc03368d66f48369b, I had put the messages under common in case they would be used in the future by another component.